README.adoc

```diff
@@ -1,36 +1,59 @@
-= Address Book (Level 4)
+= MVP (MoVie Planner)
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
-https://travis-ci.org/se-edu/addressbook-level4[image:https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master[Build Status]]
-https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
-https://coveralls.io/github/se-edu/addressbook-level4?branch=master[image:https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master[Coverage Status]]
-https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&utm_medium=referral&utm_content=se-edu/addressbook-level4&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a[Codacy Badge]]
+https://travis-ci.org/profile/CS2103JAN2018-W11-B2[image:https://travis-ci.org/CS2103JAN2018-W11-B2/main.svg?branch=master[Build Status]]
+https://ci.appveyor.com/project/tinyjy94/main[image:https://ci.appveyor.com/api/projects/status/29hd9q3b9vrvbljj/branch/master?svg=true[Build status]]
+https://coveralls.io/github/CS2103JAN2018-W11-B2/main?branch=master[image:https://coveralls.io/repos/github/CS2103JAN2018-W11-B2/main/badge.svg?branch=master[Coverage Status]]
 https://gitter.im/se-edu/Lobby[image:https://badges.gitter.im/se-edu/Lobby.svg[Gitter chat]]
 
 ifdef::env-github[]
 image::docs/images/Ui.png[width="600"]
 endif::[]
 
 ifndef::env-github[]
-image::images/Ui.png[width="600"]
+image::docs/images/Ui.png[width="600"]
```

Comment:
Please retain the images/Ui.png path.
We use scripts to populate the project list page on the module website and it picks up the image from the given path.